static/app/views/dashboards/manage/utils.tsx

```diff
@@ -3,6 +3,11 @@ import localStorage from 'sentry/utils/localStorage';
 // TODO(nikkikapadia): delete this file and move contents to index.tsx
 export const SHOW_TEMPLATES_KEY = 'dashboards-show-templates';
 
+export const WIDGET_WIDTH = 300;
```

Comment:
```suggestion
export const MINIMUM_DASHBOARD_CARD_WIDTH = 300;
```
Maybe? These aren't widgets, right? Some